js/main.js

```diff
@@ -12,15 +15,97 @@ function handlePhotoInput(event) {
 function handleSubmit(event) {
   event.preventDefault();
   const inputObj = {};
-  for (let i = 0; i < Array.from(event.target).length - 1; i++) {
-    inputObj[event.target[i].id] = event.target[i].value;
-  }
+  inputObj.title = event.target[0].value;
+  inputObj.photo = event.target[1].value;
+  inputObj.notes = event.target[2].value;
   inputObj.entryID = data.nextEntryId++;
   data.entries.unshift(inputObj);
   $form.reset();
   $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
   $photo.setAttribute('alt', 'placeholder image');
+  viewSwap('entries');
+  toggleNoEntries();
+  $entriesList.prepend(renderEntry(inputObj));
```

Comment:
You should call these before `viewSwap` so that we only switch views after the content was created